lib/rules/no-restricted-globals.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
         messages: {
             defaultMessage: "Unexpected use of '{{name}}'.",
             // eslint-disable-next-line eslint-plugin/report-message-format
-            customMessage: "Unexpected use of '{{name}}'. {{customMessage}}"
+            customMessage: "{{customMessage}}"
```

Comment:
I'm not sure if this change makes sense, why wouldn't we show the name of the restricted global variable in the message? `"Unexpected use of 'isNaN'. Use Number.isNaN instead` sounds good to me.

Reply:
Updated pull request to make this an opt-in rather than a change to the default behavior.